samples/test_i2c/src/main.c

```diff
@@ -17,10 +17,10 @@
 #include <sys/_stdint.h>
 #include <zephyr/drivers/gpio.h>
 
-/* add blinky features */
-#define SLEEP_TIME_MS   1000
-#define LED0_NODE DT_ALIAS(led0)
-static const struct gpio_dt_spec led = GPIO_DT_SPEC_GET(LED0_NODE, gpios);
+// /* add blinky features */
+// #define SLEEP_TIME_MS   1000
+// #define LED0_NODE DT_ALIAS(led0)
+// static const struct gpio_dt_spec led = GPIO_DT_SPEC_GET(LED0_NODE, gpios);
```

Comment:
Why comment this out? Instead I think you can do DT_HAS_ALIAS for an LED.

Something like:
```
#if DT_HAS_ALIAS(led0)
```

Reply:
Ok, I'll do that. Still getting up to speed on the macros.